### DISCUSSION TITLE
Volume, Network and Image management

### DISCUSSION THREAD

**Discussion by flotpg** [louislam/dockge#92] [Feature Request]
I'd love to see the possibility to list / delete volumes, networks & images created by stacks / docker.
Thanks a lot!

**Comment by spupuz** [comment]
i would love also this :)

**Comment by wildegnux** [comment]
+1 on that!
Definitely need this to properly replace portainer for my use case.

**Comment by AmIBeingObtuse** [comment]
This would be such a cool feature for this project.

**Comment by sebdanielsson** [comment]
Also looking to switch from Portainer but would like to have this feature first.

**Comment by gergepalfi** [comment]
This is a great idea, I am tired of running multiple `docker inspect <networkname>` commands just to see if things are connected. The Delete option does not remove images, volumes and networks so it would be handy for cleaning up too.

**Comment by spupuz** [comment]
@louislam any chance to get this in a future release?

**Comment by pbecks1963** [comment]
I would love to see this also!!!

**Comment by Nitrousoxide** [comment]
This would be nice, if only to graphically surface orphaned volumes, networks, and images.  Obviously you can prune them with the terminal commands, but it's nice to be able to visually see how stuff is hooked up (or not) to make sure it's all connected right.

**Comment by VladStPetre** [comment]
This would be a really useful feature

**Comment by mcdoubleyou** [comment]
that would be greate, at the moment I start Portainer or use the command to create the volumes for my containers
